crates/integration-tests/rspamd.lua

```diff
@@ -0,0 +1,102 @@
+-- Rspamd integration test policy
+-- Uses recipient address patterns to determine test behavior
+local kumo = require 'kumo'
+
+local TEST_DIR = os.getenv 'KUMOD_TEST_DIR'
+local RSPAMD_URL = os.getenv 'KUMOD_TEST_RSPAMD_URL'
+
+kumo.on('init', function()
+  kumo.start_esmtp_listener {
+    listen = '0.0.0.0:0',
+  }
+
+  kumo.start_http_listener {
+    listen = '127.0.0.1:0',
+  }
+
+  kumo.define_spool {
+    name = 'data',
+    path = TEST_DIR .. '/data-spool',
+  }
+
+  kumo.define_spool {
+    name = 'meta',
+    path = TEST_DIR .. '/meta-spool',
+  }
+end)
+
+-- Determine per-recipient spam threshold
+local function get_spam_threshold_for_user(recipient)
+  if recipient:match '@vip%.example%.com$' then
```

Comment:
Can use the `.domain` field to make this is a bit easier to consume:

```suggestion
  if recipient.domain == 'vip.example.com' then
```

https://docs.kumomta.com/reference/address/